packages/flutter/test_fixes/material.dart.expect

```diff
@@ -527,4 +527,11 @@ void main() {
 
   // Change made in https://github.com/flutter/flutter/pull/100381
   SelectionOverlay.fadeDuration;
+
```

Comment:
It looks like this is what is causing test failures here. The test functionality currently only tests the bulk applied fixes, so any `bulkApply: false` ones can't be tested here.

Reply:
I've tried both with and without bulk, but tests are still failing. I also separated removed parameter fix, no luck

Reply:
It is not possible right now to test this type of fix.

Reply:
From what I can tell, there should be one dart fix for each option of the migration with bulk apply set to false. If they are not set to bulk apply false, the dart fix tool cannot make the right decision between options and the tests fail.
With bulk apply set to false, nothing should be added to the expect files, because they only test the bulk apply fixes.